forge-gui/res/cardsfolder/upcoming/drag_to_the_roots.txt

```diff
@@ -0,0 +1,7 @@
+Name:Drag to the Roots
+ManaCost:2 B G
+Types:Instant
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ 2 | EffectZone$ All | CheckSVar$ X | SVarCompare$ GE4 | Description$ Delirium — This spell costs {2} less to cast if there are four or more card types among cards in your graveyard.

```

Comment:
`Condition$ Delirium` didn't work?

Reply:
Hm, I haven't actually tried; this code was reused from Dusk Feaster.